google-cloud-trace/docs/toc.json

```diff
@@ -1,8 +1,71 @@
 {
-  "guides": [],
+  "guides": [
+    {
+      "title": "Troubleshooting",
+      "id": "troubleshooting",
+      "edit": "https://github.com/GoogleCloudPlatform/gcloud-common/edit/master/troubleshooting/readme.md",
+      "contents": "https://raw.githubusercontent.com/GoogleCloudPlatform/gcloud-common/master/troubleshooting/readme.md"
+    },
+    {
+      "title": "Contributing",
+      "id": "contributing",
+      "edit": "https://github.com/GoogleCloudPlatform/gcloud-common/edit/master/contributing/readme.md",
+      "contents": "https://raw.githubusercontent.com/GoogleCloudPlatform/gcloud-common/master/contributing/readme.md"
+    }
+  ],
   "services": [
     {
-      "title": "TraceServiceApi",
+      "title": "Google::Cloud",
+      "type": "google/cloud"
+    },
+    {
+      "title": "Google::Cloud::Trace",
+      "type": "google/cloud/trace",
+      "nav": [
```

Comment:
Sounds good to me. I think the original concept was more of curated, guided on-boarding experience, but we should revisit that idea.